meilisearch/src/analytics/segment_analytics.rs

```diff
@@ -977,6 +988,8 @@ impl SearchAggregator {
                     "with_geoPoint": sort_with_geo_point,
                     "avg_criteria_number": format!("{:.2}", sort_sum_of_criteria_terms as f64 / sort_total_number_of_criteria as f64),
                 },
+                // TODO ask help from María
+                "distinct": distinct,
```

Comment:
```suggestion
                "distinct": distinct,
```

I guess we can remove this comment now?